### DISCUSSION TITLE
가장 많이 사용되는 position 속성 static, relative, absolute 세가지에 대해 알려주세요.

### DISCUSSION THREAD

**Discussion by javascriptbiggosoo** [wanted-9th-3team/frontend-study#48] [프론트 기술 질문 공간]
구별될만한 차이, 헷갈릴만한 부분들을 설명해주시면 됩니다.

**Comment by michoball** [comment]
static 은 HTML 요소들의 기본 값으로 아무런 위치 지정이 없는 상태를 말합니다. 
그래서 top, bottom, left, right 속성이 적용되지 않습니다. 

relative의 경우는 부모요소에 대해서 해당 요소의 위치를 상대적으로 지정할 수 있다는 뜻입니다.  relative로 지정할 경우 앞서 말한 요소들로 상대적인 위치를 지정할 수 있습니다. 

absolute의 경우는 부모 요소 중 가장 가까운 postion 값이 relative 또는 absolute인 요소를 기준으로 위치를 지정하겠다는 뜻입니다. 만약 부모요소가 static 값이면 동작하지 않습니다.

**Comment by WooGie911** [comment]
static  :
기본값으로 모든 태그들은 position: static 상태입니다.
보통 따로 위치를 지정하지 않을 때 사용합니다.

relative : 
위치를 계산할때 static의 원래 위치부터 계산합니다.
top, right, bottom, left 방향으로 주어진 픽셀만큼 이동 할 때 사용합니다.

absolute : 
원래(position: static 속성을 가지고 있는)위치와 상관없는 위치를 지정할 수 있습니다. 
단, 가장 가까운 상위 요소를 기준으로 위치가 결정 됩니다.


이 외에도 fixed 가 있는데 fixed 는 원래 위치와 상관없이 위치를 지정할 수 있습니다. 
하지만 상위 요소에 영향을 받지 않기 때문에 스크롤을 내리는 등 화면이 바뀌더라도 위치를 고정하는 설정 할 수 있습니다.

**Comment by javascriptbiggosoo** [comment]
쓰다보니 길어져서 블로그에다 정리해봤습니다.
https://velog.io/@tbs01215/position-%EA%B7%B8%EB%A7%8C-%ED%97%B7%EA%B0%88%EB%A6%AC%EC%9E%90

**Comment by BaekYuri** [comment]
static : position 속성의 default 값.
relative: static의 원래 위치 값부터 계산하고, top, bottom, left, right 속성을 이용해 원래 위치를 기준으로 어떻게 움직일지 설정할 수 있음
absolute: 부모 중에 static이 아닌 요소를 기준으로 top, bottom, left, right 속성으로 위치 정할 수 있음
